### PR TITLE
DBotTrainTextClassifierV2: error messages & limiting min incidents

### PR DESCRIPTION
**Pull request by lperryp**
## Description
1. making the error messages clearer
2. assigning a threshold (currently = 5) for minimum number of incidents
3. changing map functions as preparation for transitioning to python 3

## Status
Ready

##Related Issues
https://github.com/demisto/etc/issues/21211

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by kirbles19** [comment]
@lperryp Done.

File: Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 }
 
 DBOT_TAG_FIELD = "dbot_internal_tag_field"
+MIN_INCIDENTS_THRESHOLD = 5
```
Comment (erezh31) [inline Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py]: make it 50, for each type

File: Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 2]
```diff
@@ -245,6 +259,19 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
...
         demisto.results(entry)
+    demisto.results(set([x[DBOT_TAG_FIELD] for x in data]))
+    if len(set([x[DBOT_TAG_FIELD] for x in data])) == 1:
+        single_label = [x[DBOT_TAG_FIELD] for x in data][0]
+        err = ['All received incidents have the same label: {}.'.format(single_label)]
+        err += ['At least 2 different labels are required to train a classifier.']
+        if labels_mapping == ALL_LABELS:
+            err += ['Please make sure that incidents of at least 2 labels exist in the environment.']
+        else:
+            unfound_mapped_label = [l for l in labels_mapping if l not in exist_labels_counter
```
Comment (erezh31) [inline Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py]: This message is for label that the user define but does not exist in the data probably a mistake of the user

File: Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 3]
```diff
@@ -245,6 +259,19 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
...
+                                    or exist_labels_counter[l] == 0]
```
Comment (erezh31) [inline Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py]: Add here also missing_labels_counter for labels that were not mapped

**Pull request merged**